add information on disabling local repositories (#1660)
uyuni-project/uyuni-docs: modules/client-configuration/pages/registration-webui.adoc

To register and use {centos}{nbsp}6, {oracle}{nbsp}6, {rhel}{nbsp}6, or {sleses}{nbsp}6 clients, you need to configure the {productname} Server to support older types of SSL encryption.
For more information, see ``Registering Older Clients`` at xref:client-configuration:tshoot-clients.adoc[].
====

=== Handling of Locally assigned Repositories

> Having repositories assigned directly to clients not served by {productname} is not usual and can create problems.

Reviewer: ```suggestion
Having repositories assigned directly to clients not served by {productname} is not a common use case.
It can cause trouble.
```

> Therfore bootstrapping via Salt disable all local repositories at the beginning of the bootstrap process.

Reviewer: ```suggestion
Therfore bootstrapping via Salt disables all local repositories at the beginning of the bootstrap process.
```

> Also later on every use of the channel state, for example when executing a Highstate or a Package installation,
> all locally assigned repositories are disabled again.

Reviewer: ```suggestion
Later, during every use of the channel state, for example when executing a Highstate or a package installation, all locally assigned repositories are disabled again.
```

All software packages which are used on the clients should come from channels served by {productname}.

> For more informationon how to create custom channel, see ``Custom Channels`` at xref:administration:custom-channels.adoc[].

Reviewer: ```suggestion
For more information about creating a custom channel, see ``Custom Channels`` at xref:administration:custom-channels.adoc[].
```
